src/executables/util_cs.c

```diff
@@ -3235,6 +3242,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    }
 	}
     }
+  if ((!check_copied_info) && (!check_applied_info) && (!master_node_name) && (pageid == APPLYINFO_DEFAULT_LOG_PAGEID))
+    {
+      goto print_applyinfo_usage;
+    }
```

Comment:
What is it for? If no option is given, is the utility supposed to fail? Is there no default behavior or output?

Reply:
This command prints nothing.`cubrid applyinfo demodb`
And there is no fail message. So I set it as usage.

Reply:
Aha, I see. Good Job.
However, I think users don't know how to deal with the error with the usage because the usage doesn't mention one of them has to be given. So, I think the usage and the manual should be revised together.

And -a, -p requires -L and it's dealt with in another place. right? Then, I think the following is enough:
```suggestion
  if ((!check_copied_info) && (!master_node_name))
    {
      goto print_applyinfo_usage;
    }
```